backend/src/graphql/resolver/util/authenticateGmsUserPlayground.ts

```diff
@@ -7,7 +7,7 @@ import { backendLogger as logger } from '@/server/logger'
 import { ensureUrlEndsWithSlash } from '@/util/utilities'
 
 export async function authenticateGmsUserPlayground(
-  apiKey: string,
+  _apiKey: string,
```

Comment:
whats the motivation to set a leading underscore?
normally the apikey should be used as parameter for the gms-request, but currently the authentication is deactivated. That's the reason why the apiKey isn't in use here.

Reply:
> Disallow unused variables.

> There is an exception to this rule: variables that starts with underscore, e.g. let _something;.

> The pattern of having an underscore as prefix of a name of variable is a very diffuse pattern among programmers, and Biome decided to follow it.

https://biomejs.dev/linter/rules/no-unused-variables/
https://biomejs.dev/linter/rules/no-unused-function-parameters/

It is to mark the variable as intentional unused.